src/core/common/instance.hpp

```diff
@@ -326,6 +327,16 @@ class Instance : public otInstance, private NonCopyable
     Coap::CoapSecure &GetApplicationCoapSecure(void) { return mApplicationCoapSecure; }
 #endif
 
+#if OPENTHREAD_CONFIG_BLE_TCAT_ENABLE
+    /**
+     * This method returns a reference to application BLE Secure object.
+     *
+     * @returns A reference to the application BLE Secure object.
+     *
+     */
+    Ble::BleSecure &GetApplicationBleSecure(void) { return mApplicationBleSecure; }
```

Comment:
We can simplify this and use `Get<Ble::BleSecure>` instead?
I don't think we will have multiple `Ble::BleSecure` instances in an `ot::Instance`? Is this right?

Reply:
fixed